syntax = "proto2";
package spider;

message produce_url_resp {
	required int32 res = 1;
}
